mne/cov.py

```diff
@@ -801,6 +801,8 @@ def _unpack_epochs(epochs):
         data_mean = [1.0 / n_epoch * np.dot(mean, mean.T) for n_epoch, mean
                      in zip(n_epochs, data_mean)]
 
+    if info['comps']:
+        info['comps'] = []
```

Comment:
this does an in-place modification of the `info`. Is this desired? `compute_covariance` should not change `epochs.info` IMO. It led me to a long debugging session with a colleague ...

Reply:
That does sound like a bug, possibly avoidable with `info = info.copy()` in a line above